src/explore-education-statistics-frontend/src/modules/subscriptions/VerificationErrorPage.tsx

```diff
@@ -0,0 +1,26 @@
+import Page from '@frontend/components/Page';
+import { NextPage } from 'next';
+import React from 'react';
+
+const VerificationErrorPage: NextPage = () => {
+  return (
+    <Page title="Verification failed">
+      <div className="govuk-panel__body">
+        <p>
+          Your subscription verification token has expired. You can try again by
+          re-subscribing from the publication's main screen.
```

Comment:
This may be scope creep, but it might be a nicer user journey if we were to make `publication's main screen` a link to said main screen. Could we send the publication id in the RedirectResult and add it as a prop here?

Reply:
I considered this, but the data isn't available at this stage because the token linking a user to a publication has been deleted. Mentioned to Cam and he's happy with current messaging, and can iterate down the line if it's deemed worth the effort investigating.